fastembed/embedding.py

```diff
@@ -144,18 +166,17 @@ def __init__(
             max_length (int, optional): The maximum length of the input text. Defaults to 512.
 
         Raises:
-            ValueError: If the model_name is not in the format <org>/<model> e.g. BAAI/bge-base-en.    
+            ValueError: If the model_name is not in the format <org>/<model> e.g. BAAI/bge-base-en.
         """
 
-        assert "/" in model_name, "model_name must be in the format <org>/<model> e.g. BAAI/bge-base-en"
-        model_name = model_name.split("/")[-1]
-        fast_model_name = f"fast-{model_name}"
-        filepath = self.download_file_from_gcs(
-            f"https://storage.googleapis.com/qdrant-fastembed/{fast_model_name}.tar.gz",
-            output_path=f"{fast_model_name}.tar.gz",
-        )
-        model_dir = self.decompress_to_cache(targz_path=filepath)
-        model_dir = Path(model_dir) / fast_model_name
+        if onnx_providers is None:
+            onnx_providers = [ONNXProviders.CPU]
```

Comment:
I prefer having this in the function prototype because then the default is clear and so is the expected e.g. `List[...]` here

Reply:
I agree that it looks better, but in python default values are mutable. So even PyCharm considers it a bad practice.